test/integration/MathMax/math max.cs

```diff
@@ -0,0 +1,5 @@
+-
+using System;
+
+Math.Max(foo, bar);
+-
```

Comment:
These files should have an endline so there's no little red symbol at the end in the PR.

Added a mention to #185.